docs/modules/ROOT/pages/testing.adoc

```diff
@@ -0,0 +1,171 @@
+ifndef::backend-pdf[]
+:examplesdir: example$
```

Comment:
This makes sense when using live code that is maintained and used in files on their own. For this case of isolated documentation. I suggest to directly embed it into the document.